src/components/icons/edit.tsx

```diff
@@ -1,5 +1,5 @@
 /*!
- * © 2019 Atypon Systems LLC
+ * © 2024 Atypon Systems LLC
```

Comment:
why is this changed only in some file and in some it's not?

Reply:
This is actually a new file, but for some reason git is seeing it as a rename of another file that I deleted "bookmark".